src/plasmapy/diagnostics/charged_particle_radiography/synthetic_radiography.py

```diff
@@ -201,16 +206,17 @@ def __init__(
             "volume averaged", "nearest neighbor"
         ] = "volume averaged",
         detector_hdir=None,
-        output_file: Path | None = None,
+        output_directory: Path | None = None,
+        output_name: str = "output",
         fraction_exited_threshold: float = 0.999,
         verbose: bool = True,
     ) -> None:
         # The synthetic radiography class handles logging, so we can disable logging for the particle tracker
         # The particle tracker class ensures that the provided grid argument has the proper type and
         # that the necessary grid quantities are created if they are not already specified
         save_routine = (
-            _SyntheticRadiographySaveRoutine(output_file)
-            if output_file is not None
+            _SyntheticRadiographySaveRoutine(output_directory, output_name)
+            if output_directory is not None
```

Comment:
🤔 Would it make sense to combine `output_directory` and `output_name` to a single `output_path` parameter?

Reply:
I tried this, the problem is that this can be a directory (if multiple output files are created, e.g. at multiple time steps) or a single file (for a single output file at the end of a simulation). So if this is just one `path` keyword, I would have to depend on looking at file extensions or something to figure out which it was...

Reply:
Makes sense!  I updated my comment around the same time you posted this.

Reply:
Yes good call, I like output_basename: change made